tasks/test-coverage.js

```diff
@@ -87,9 +73,9 @@ var setupBackupAndInstrumentationDir = function() {
   }
 
   log('• copy src files to backup folder');
-  wrench.copyDirSyncRecursive(dir, backupDir, copyOpts);
+  fse.copySync(dir, backupDir);
```

Comment:
This wont overwrite existing files does that matter? Also preserving timestamps? ```{clobber: true, preserveTimestamps: true}```